defs.h

```diff
@@ -46,8 +46,9 @@
 #include "geocache.h"                // for Geocache
 #include "formspec.h"                // for FormatSpecificData
 #include "inifile.h"                 // for inifile_t
-#include "option.h"
 #include "session.h"                 // for session_t
+
+class Option;                      // for arglist_t
```

Comment:
comment should be // forward declare
I would suggest moving it after all the #includes.